pkgs/top-level/all-packages.nix

```diff
@@ -1166,6 +1166,10 @@ with pkgs;
 
   yaft = callPackage ../applications/terminal-emulators/yaft { };
 
+  abricotine = callPackage ../applications/editors/abricotine {
+    electron = electron_6;
```

Comment:
This version seems to be removed from nixpkgs:
```shell
~ > nix eval nixpkgs\#electron_6
error: electron_6 has been removed in favor of newer versions
```
Could we use a more recent version?

Reply:
No, that was the version that made it work. Further versions didn't show anything on the interface.

I will close the PR, because the application hasn't brought up new versions.